crest/Assets/Crest/Crest/Scripts/Spline/SplinePoint.cs

```diff
@@ -0,0 +1,223 @@
+﻿// Crest Ocean System
+
+// This file is subject to the MIT License as seen in the root of this folder structure (LICENSE)
+
+using UnityEditor;
+using UnityEngine;
+
+namespace Crest.Spline
+{
+    /// <summary>
+    /// Spline point, intended to be child of Spline object
+    /// </summary>
+    [ExecuteAlways]
+    public class SplinePoint : MonoBehaviour
+    {
+#if UNITY_EDITOR
+        Spline _spline;
+
+        void Awake()
+        {
+            _spline = transform.parent != null ? transform.parent.GetComponent<Spline>() : null;
+        }
```

Comment:
This is no longer referenced anywhere so it can be removed.

Reply:
oh nice, done